sky/templates/paperspace-ray.yml.j2

```
@@ -33,6 +33,7 @@ provider:
 auth:
   ssh_user: paperspace
   ssh_private_key: {{ssh_private_key}}
+  ssh_public_key: skypilot:ssh_public_key_content
```

It would be better to use `|-` to treat the public key content as literal, to avoid any unexpected parsing issue.
```suggestion
  ssh_public_key: |-
      skypilot:ssh_public_key_content
```